go/engine/pgp_test.go

```diff
@@ -38,57 +39,57 @@ func TestPGPUserInterface(t *testing.T) {
 	p := newPgpPair(t)
 	defer p.cleanup()
 
-	p.assert(!p.senderTracksRecipient(), "sender shouldn't track recipient")
-	p.assert(!p.recipientTracksSender(), "recipient shouldn't track sender")
+	require.True(p.t, !p.senderTracksRecipient(), "sender shouldn't track recipient")
+	require.True(p.t, !p.recipientTracksSender(), "recipient shouldn't track sender")
 
 	// encrypt, not signed
 	notSigned, idCount := p.encrypt(false)
 
 	// test that identify ui was shown once to sender for recipient
-	p.assertEqual(idCount, 1, "identify ui count [encrypt, not signed]")
+	require.Equal(p.t, idCount, 1, "identify ui count [encrypt, not signed]")
 	// with identify2, no tracking, so test that sender doesn't track recipient
-	p.assert(!p.senderTracksRecipient(), "after encrypt, sender shouldn't track recipient")
-	p.assert(!p.recipientTracksSender(), "after encrypt, recipient shouldn't track sender")
+	require.True(p.t, !p.senderTracksRecipient(), "after encrypt, sender shouldn't track recipient")
+	require.True(p.t, !p.recipientTracksSender(), "after encrypt, recipient shouldn't track sender")
 
 	// encrypt, signed
 	signed, idCount := p.encrypt(true)
 
 	// test that identify ui was *not* shown to sender for recipient since
 	// just shown in previous p.encrypt() call.
-	p.assertEqual(idCount, 0, "identify ui count [encrypt, signed]")
+	require.Equal(p.t, idCount, 0, "identify ui count [encrypt, signed]")
 	// with identify2, not tracking, so test that sender doesn't track recipient
-	p.assert(!p.senderTracksRecipient(), "after encrypt, sender shouldn't track recipient")
-	p.assert(!p.recipientTracksSender(), "after encrypt, recipient shouldn't track sender")
+	require.True(p.t, !p.senderTracksRecipient(), "after encrypt, sender shouldn't track recipient")
+	require.True(p.t, !p.recipientTracksSender(), "after encrypt, recipient shouldn't track sender")
 
 	// decrypt, not signed
-	p.checkIdentifyUIAndPgpUI("decrypt not signed", p.decrypt, notSigned, 0)
+	p.checkIdentifyUIAndPgpUI("decrypt not signed", p.decrypt, notSigned, 0, 0)
```

Comment:
The `checkIdentifyUIAndPgpUI` this diff changed are the ones with `0`s after the above.